### DISCUSSION TITLE
I could not disable the web context menu using your article.

### DISCUSSION THREAD

**Discussion by bambinoua** [nank1ro/flutter-shadcn-ui#207] [General]
I tried to use your article https://mariuti.com/posts/flutter-web-disable-browser-context-menu-for-specific-widget/ but unfortunately the described approach is not working. I tested it and the context menu still visible. Do you have any thoughts on this?

**Comment by nank1ro** [comment]
In which platform are you trying? Windows?

Can you clarify if you're using the ShadContextMenu widget or just following the approach described in the article for another widget?

**Comment by bambinoua** [comment]
I use exactly the approach describe in the article. Basically I need to disable context menu _**only for specific widgets**_ and **_NOT_** for the whole window.

**Reply by nank1ro** [reply]
But in which platform do you experience the bug?

**Comment by bambinoua** [comment]
> But in which platform do you experience the bug?

Yes, Windows.

P.S. I really didn't understand why the question of platform if your article describes the way to disable **_web context menu_**?

**Reply by nank1ro** [reply]
Because on Windows the webContextMenu is not shown on tapDown, but on tapUp instead.
Someone raised an issue about it, I fixed it here on shadcn_ui but I haven't updated the article.
I was just ensuring you were on Windows and haven't encountered another different issue.

Let me revise the article

**Reply by nank1ro** [reply]
@bambinoua Can you try on this web app https://987d6bac.disable-web-context-menu.pages.dev/ ?

I tried on both Mac and Windows and inside the container the native context menu is not shown.
If it works I'm going to release a package.

**Reply by nank1ro** [reply]
this is the code https://github.com/nank1ro/disable_web_context_menu

**Reply by bambinoua** [reply]
> @bambinoua Can you try on this web app https://987d6bac.disable-web-context-menu.pages.dev/ ?

Looks working.

**Reply by nank1ro** [reply]
Just released the package https://pub.dev/packages/disable_web_context_menu
